arithmetization/src/main/java/net/consensys/linea/zktracer/module/tables/shf/Trace.java

```diff
@@ -184,7 +184,7 @@ public Trace fillAndValidateRow() {
     }
 
     if (!filled.get(3)) {
-      mshp.position(mshp.position() + 2);
+      mshp.position(mshp.position() + 1);
```

Comment:
What does this do ?
I imagine that the 2 -> 1 here is the same as the 2 -> 1 change in the ColumnHeader for MSHP ?

Reply:
yes, its is. Generated by corset.